test/controllers/organization_controller_test.exs

```diff
@@ -26,12 +26,9 @@ defmodule CodeCorps.OrganizationControllerTest do
       path = "organizations/?filter[id]=#{first_org.id},#{second_org.id}"
       conn = conn |> get(path)
 
-      data = json_response(conn, 200)["data"]
+      response = json_response(conn, 200)
 
-      [first_result, second_result | _] = data
-      assert length(data) == 2
-      assert first_result["id"] == "#{first_org.id}"
-      assert second_result["id"] == "#{second_org.id}"
+      assert ids_from_response(response) == [first_org.id, second_org.id]
```

Comment:
Just a suggestion, but how about writing an `assert_ids_from_response/2` helper and then rewriting this as 

```elixir
conn
|> get(path)
|> json_response(200)
|> assert_ids_from_response([first_org.id, second_org.id])
```

We already have an `assert_response_id` helper that works similarly.

Thoughts?